core/domain/platform_parameter_domain.py

```diff
@@ -28,11 +28,14 @@
 from core.constants import constants
 from core.domain import change_domain
 
+from typing import Any, Dict
 
 # TODO(#14419): Change naming style of Enum class from SCREAMING_SNAKE_CASE
 # to PascalCase and its values to UPPER_CASE. Because we want to be consistent
 # throughout the codebase according to the coding style guide.
 # https://github.com/oppia/oppia/wiki/Coding-style-guide
+
+
```

Comment:
Why is this added?

Reply:
because not doing this it gives me lint error.

Reply:
You need to add the line above.

Reply:
Done!